Frameworks/include/CGIWICBitmap.h

```diff
@@ -40,6 +40,8 @@
 #import <D2d1.h>
 #include <COMIncludes_End.h>
 
+#import <memory.h>
+
 using namespace Microsoft::WRL;
```

Comment:
more of the old code. Agreed.

---
In reply to: [88099822](https://github.com/Microsoft/WinObjC/pull/1371#discussion_r88099822) [](ancestors = 88099822)